packages/rx_bloc_list/lib/src/widgets/rx_paginated_builder.dart

```diff
@@ -194,14 +239,35 @@ class _RxPaginatedBuilderState<B extends RxBlocTypeBase, T>
           bloc: bloc,
           state: widget.state,
           builder: (context, snapshot, bloc) {
-            final builtWidget = widget.builder(context, snapshot, bloc);
+            final buildChild = _buildChild(context, snapshot, bloc);
+
             if (widget.wrapperBuilder != null)
-              return widget.wrapperBuilder!.call(context, bloc, builtWidget);
-            return builtWidget;
+              return widget.wrapperBuilder!.call(context, bloc, buildChild);
+            return buildChild;
           }),
     );
   }
 
+  Widget _buildChild(
+    BuildContext context,
+    AsyncSnapshot<PaginatedList<T>> snapshot,
+    B bloc,
+  ) {
+    if (!snapshot.hasData) {
```

Comment:
Shouldn't we treat this as a loading state too? Which are the cases where we would prefer having an empty container instead of a loading widget?

Reply:
Done